lib/measure_repository_service_test_kit/patient_group.rb

```diff
@@ -1,4 +1,7 @@
-module InfernoTemplate
+# frozen_string_literal: true
+
+module MeasureRepositoryServiceTestKit
+  # Example Patient group test suite
```

Comment:
we should probably just delete this right? It's going to get immediately deleted anyway when we do our first task. It's fine to just leave the capability statement test in and get rid of the patient group

Reply:
Sounds good! I wasn't sure if we wanted to keep an example or anything